wiki/Tournaments/o!cCC/1/en.md

```diff
@@ -0,0 +1,234 @@
+
+- NoMod
+  - [Unlucky Morpheus - Phantom Blood (IsomirDiAngelo) \[Atalanta's Extra\]](https://osu.ppy.sh/beatmapsets/885158#fruits/2153771)
+  - [Coda - BLOODY STREAM (Shiguma) \[OVERDRIVE\]](https://osu.ppy.sh/beatmapsets/674882#fruits/1480932)
+  - [twenty one pilots - Levitate (Mao) \[Vulture\]](https://osu.ppy.sh/beatmapsets/1026205#fruits/2146132)
+  - [Akira Complex - Ether Strike (Mun) \[Mir's Aethereal Expert\]](https://osu.ppy.sh/beatmapsets/945019#fruits/1973467)
+- Hidden
+  - [Rammstein - Feuer frei! (Frey) \[Fire at will! BANG! BANG!!\]](https://osu.ppy.sh/beatmapsets/887102#fruits/1854467)
+  - [Gesu no Kiwami Otome. - DARUMASAN (celerih) \[dsco's EXTREME\]](https://osu.ppy.sh/beatmapsets/613613#fruits/1302779)
+  - [Grand Thaw - Aventyr (kwk) \[Adventure\]](https://osu.ppy.sh/beatmapsets/556922#fruits/1178554)
+- HardRock
+  - [Morimori Atsushi+Ujico\* - Vieille Ville (cheesiest) \[diff\]](https://osu.ppy.sh/beatmapsets/484162#fruits/1032837)
+  - [Camellia - Diastrophism (Regou) \[Insane\]](https://osu.ppy.sh/beatmapsets/923095#fruits/1932099)
+  - [Glutamine - Babylon (CSY the corrupt) \[Corruption\]](https://osu.ppy.sh/beatmapsets/35992#fruits/145118)
+- DoubleTime
+  - [Katy Perry - E.T. (La Cataline) \[Futuristic\]](https://osu.ppy.sh/beatmapsets/36077#fruits/116708)
+  - [Dj Verstarker - Jump Jump (2008) (SeymourLyon) \[Jump like CRAZY!!!\]](https://osu.ppy.sh/beatmapsets/6004#fruits/28152)
+  - [Sharlo - Shinkai Shoujo (yf\_bmp) \[Insane\]](https://osu.ppy.sh/beatmapsets/221768#fruits/518731)
+- Tiebreaker
+  - **[Feryquitous - Yoi no Tamoto, Yoru to Futari (Ametrin) \[Monogatari\]](https://osu.ppy.sh/beatmapsets/891830#fruits/1864425)**
+
+### Qualifiers Stage
+
+**[Download the mappack here! (25 MB)](https://mega.nz/#!sodzSKQJ!1_UNqV2Em-bnD1yK6aQWFqnYryLGiNtXXFxiA9_DXTc)**
+
+- NoMod
+  - [Laura Bodewig (CV : Inoue Marina) - SUPER STREAM (Akitoshi) \[Next Future\]](https://osu.ppy.sh/beatmapsets/1039295#fruits/2172460)
+  - [L.E.D.-G - MVA (ouranhshc) \[Collab\]](https://osu.ppy.sh/beatmapsets/597898#fruits/1263892)
+- Hidden
+  - [Rin - Prism Magical (DJ SHARPNEL hardrave remix) (regenz) \[Extra Magical\]](https://osu.ppy.sh/beatmapsets/39032#fruits/204536)
+- HardRock
+  - [Kola Kid - can't hide your love (Kert) \[Can\]](https://osu.ppy.sh/beatmapsets/39732#fruits/126470)
+- DoubleTime
+  - [VY1 - Cyber Thunder Cider (tutuhaha) \[Cyber\]](https://osu.ppy.sh/beatmapsets/47146#fruits/151215)
+
+---
+
+## Match Results
+
+### Round of 16
+
+| Saturday, 2019-10-05 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+
+| Sunday, 2019-10-06 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+
+### Round of 32
+
+| Saturday, 2019-09-28 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| Aryssiel ![][flag_CL] | 0 | **4** | ![][flag_PH] **Jemzuu** | -win by default- |
+| chickennando ![][flag_AU] | 1 | **4** | **Galaxi** ![][flag_ID] | [#1](https://osu.ppy.sh/community/matches/55160792) |
```

Comment:
galaxi flag backwards

if the flags aren't particularly important I think it might be a good idea to remove them from this section. they aren't usually in match results, right?

Reply:
I remember them being in most match results' wiki of my tournaments so uhh

Reply:
yeah looks like I was just remembering an outlier. edited comment